GDJS/GDJS/IDE/ExporterHelper.cpp

```diff
@@ -115,7 +116,8 @@ bool ExporterHelper::ExportProjectForPixiPreview(
                  immutableProject.GetLoadingScreen().GetGDevelopLogoStyle(),
                  includesFiles);
 
-  // Export files for object and behaviors
+  // Export files for free function, object and behaviors
+  ExportFreeFunctionIncludes(exportedProject, includesFiles);
```

Comment:
You also need to add this in `ExportWholePixiProject` (yes, this should be factored at some point)